ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,4616 @@
+        )
+        try:
+            self.patch_storagecluster_cephx_component(
+                self.COMPONENT_DAEMON, component_config
+            )
+        except CommandFailed as exc:
+            err = str(exc)
+            if constants.CEPHX_KEY_GENERATION_DECREASE_ERROR not in err:
+                raise UnexpectedBehaviour(
+                    "Expected StorageCluster admission error containing "
+                    f"'{constants.CEPHX_KEY_GENERATION_DECREASE_ERROR}', got: {err}"
+                )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)